### PR TITLE
[FEM] fix detached plot

### PR DESCRIPTION
**Pull request by donovaly**
the stress linearization plot is at the moment detached from FreeCAD's main window.
This PR fixes it the same way as in https://github.com/FreeCAD/FreeCAD/commit/920e8e046

### CODE REVIEW

**Comment by donovaly** [comment]
@0penBrain , can you please have a look? I used your metod to fix the stress linearization window. It works for me as it should but since I am on Win I would like to get another test.

The plot is this one: https://wiki.freecadweb.org/FEM_PostFilterLinearizedStresses

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6998">pipeline status</a> for feature branch [PR_6998](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6998). Pipeline [559657482](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/559657482) was triggered at [e4e9d2e](https://github.com/FreeCAD/FreeCAD/pull/6998/commits/e4e9d2e9cc21e18e02317a931548b850e477b363). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
#6999 uses the same approach as this PR

File: src/Mod/Fem/Gui/Command.cpp [hunk 1]
```diff
@@ -1249,6 +1249,8 @@ for i in range(len(sValues)):\n\
     peak.append(sValues[i])\n\
     mb.append(bending[i] + membrane[0])\n\
 import FreeCAD\n\
+from PySide import QtCore\n\
+from PySide.QtCore import Qt\n\
```
Comment (0penBrain) [inline src/Mod/Fem/Gui/Command.cpp:1253]: This line isn't needed.

**Review by 0penBrain — COMMENTED** [review]
@donovaly I'm sorry I can't test ATM, so I just did a code review.

File: src/Mod/Fem/Gui/Command.cpp (hunk 1 above)
Comment (donovaly) [reply src/Mod/Fem/Gui/Command.cpp:1253]: Indeed, thanks.

**Pull request merged**